trending_data/src/main/webapp/read_maps_key.js

```diff
@@ -0,0 +1,19 @@
+const file = new XMLHttpRequest();
+file.open('GET', 'key.txt', false);
+
+file.onreadystatechange = function() {
+  if (file.readyState === 4) {
+    if (file.status === 200 || rawFile.status == 0) {
+      const $key = file.responseText;
+
+      const mapsAPIScript = 'https://maps.googleapis.com/maps/api/js?key=' + $key + '&callback=initMap&libraries=&v=weekly';
+
+      const mapScriptTag = document.createElement('script');
+      mapScriptTag.src = mapsAPIScript;
+      mapScriptTag.type = 'text/javascript';
+
+      document.getElementsByTagName('head')[0].appendChild(mapScriptTag);
```

Comment:
I think you can just access `document.head` instead of `document.getElementsByTagName('head')[0]`, and you don't need to explicitly set an ID for the <head> element in HTML.

Reply:
I forgot to delete the id attribute, It was not used anyway because I am getting the element by tag name rather than id.

Reply:
Yes, `document.head` works!